scripts/lib/setup/installers.sh

```diff
@@ -170,9 +180,25 @@ function install_android_sdk() {
 }
 
 function install_android_sdk_linux() {
-  cecho "@b@yellow[[+ Skipping Android SDK setup, not implemented on Linux]]"
-
-  return 1
+  if [ -e ~/Android ]; then
```

Comment:
Same thing here with `$ANDROID_SDK_ROOT`, we could also test if a directory exists at `$ANDROID_SDK_ROOT` if it is defined.